cocos/particle-2d/motion-streak-2d-assembler.ts

```diff
@@ -244,6 +243,10 @@ export const MotionStreakAssembler: IAssembler = {
     },
 
     updateRenderData (comp: MotionStreak) {
+        if (JSB) {
```

Comment:
Could you please add some comments to this code so that it can be identified for subsequent maintenance: like this

```
// A dirty hack
// The world matrix was updated in advance and needs to be avoided at the cpp level
// Need a flag to explicitly not update the world transform to solve this problem
```

Reply:
Thanks for the explanation. I added the comments.